Documents/MobileAppActors.txt

```diff
@@ -0,0 +1,78 @@
+Actors: User
```

Comment:
Try thinking which of it could be done by a guest (someone without logging in) and which of it would be done by a user (probably via google play account).

You don't need to login in order to draw.

However you do need that if you want to give likes. Publishing drawings could be done by anyone, with admins approval

Reply:
I've redone this part, so now users need to sign in or continue as guests whilst using the app. Both roles have pretty much the exact same options, except "Guests" cannot like artwork in galleries (but can still publish with admin approval).